hack/test-cmd.sh

```diff
@@ -136,6 +136,11 @@ KUBELET_HEALTHZ_PORT=${KUBELET_HEALTHZ_PORT:-10248}
 CTLRMGR_PORT=${CTLRMGR_PORT:-10252}
 PROXY_HOST=127.0.0.1 # kubectl only serves on localhost.
 
+IMAGE_NGINX="gcr.io/google-containers/nginx:1.7.9"
+IMAGE_DEPLOYMENT_R1="nginx"  # controlled by docs/user-guide/deployment.yaml
```

Comment:
Since docs/ is removed to the doc repo and isn't maintained here anymore. Can you create a deployment yaml in hack/testdata and use the gcr image instead?